multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -475,7 +533,7 @@ def infer_label_column_type_by_problem_type(
     problem_type: Optional[str],
     data: Optional[pd.DataFrame] = None,
     valid_data: Optional[pd.DataFrame] = None,
-    allowable_label_types: Optional[List[str]] = (CATEGORICAL, NUMERICAL, NER_ANNOTATION, ROIS),
+    allowable_label_types: Optional[List[str]] = (CATEGORICAL, NUMERICAL, NER_ANNOTATION, ROIS, DOCUMENT),
```

Comment:
Is document also used as label? It seems that we only use it as input for now.

Reply:
removed, thanks for pointing out.